docs/Admin.md

```diff
@@ -931,7 +931,7 @@ const knownResources = [
     <Resource name="comments" list={CommentList} />,
 ];
 
-const fetchResources = permissions =>
+const fetchResources = (permissions: string) =>
```

Comment:
`string` is a weird type for permissions IMO. I'd go for `string[]` instead, or even `any` since they can actually be of any type

Reply:
permissions is any